python/ray/autoscaler/_private/kuberay/autoscaling_config.py

```diff
@@ -108,11 +110,18 @@ def _derive_autoscaling_config_from_ray_cr(ray_cr: Dict[str, Any]) -> Dict[str,
     if IDLE_SECONDS_KEY in autoscaler_options:
         idle_timeout_minutes = autoscaler_options[IDLE_SECONDS_KEY] / 60.0
     else:
-        idle_timeout_minutes = 5.0
-    if autoscaler_options.get(UPSCALING_KEY) == UPSCALING_VALUE_AGGRESSIVE:
-        upscaling_speed = 1000  # i.e. big
+        idle_timeout_minutes = 1.0
+
+    if autoscaler_options.get(UPSCALING_KEY) == UPSCALING_VALUE_CONSERVATIVE:
+        upscaling_speed = 1  # Rate-limit upscaling if "Conservative" is set by user.
+    # This elif is redudant but included for clarity.
+    elif autoscaler_options.get(UPSCALING_KEY) == UPSCALING_VALUE_DEFAULT:
+        upscaling_speed = 1000  # i.e. big, no rate-limiting by default
```

Comment:
I assume people know about this change and its implications?

Reply:
People can refer to two groups

Users:
Autoscaling support is new in KubeRay --- it suffices to cover in the docs, which I will do.

Ray developers:
Are aware of the change and its implications.